src/main/java/org/opentripplanner/openstreetmap/model/OSMNode.java

```diff
@@ -59,6 +60,19 @@ public boolean isBarrier() {
     );
   }
 
+  /**
+   * Consider barrier tag in  permissions. Leave the rest for the super class.
+   */
+  @Override
+  public StreetTraversalPermission overridePermissions(StreetTraversalPermission def) {
+    StreetTraversalPermission permission = def;
+    if (isBollard()) {
```

Comment:
Btw it's a bit weird that the barrier tag can have a lot of values but we have this special handling for bollard.

Reply:
Good point. I will add some more.